docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -174,12 +172,12 @@ The following instruction are written generically so they can be used in multipl
         timesGetOneDriveFilesHasRun++;
         triedWithoutForceConsent = true;
         getDataWithToken({ forceConsent: false });
-    }	
+    }
     ```
 
 1. Below the `getOneDriveFiles` method, add the code below. Note the following about this code:
 
-    * The [getAccessTokenAsync](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins#sso-api-reference) is the new API in Office.js that enables an add-in to ask the Office host application (Excel, PowerPoint, Word, etc.) for an access token to the add-in (for the user signed into Office). The Office host application, in turn, asks the Azure AD 2.0 endpoint for the token. Since you preauthorized the Office host to your add-in when you registered it, Azure AD will send the token.
+    * The [getAccessTokenAsync](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins#sso-api-reference) is the new API in Office.js that enables an add-in to ask the Office host application (Excel, PowerPoint, Word, etc.) for an access token to the add-in (for the user signed into Office). The Office host application, in turn, asks the Azure AD 2.0 endpoint for the token. Since you pre-authorized the Office host to your add-in when you registered it, Azure AD will send the token.
```

Comment:
Please change "pre-authorized" back to "preauthorized". I don't think any of the circumstances that would require a hyphen after a prefix apply in this case.